news/9647.doc.rst

```diff
@@ -0,0 +1 @@
+Add warning about --extra-index-url and dependency confusion
```

Comment:
```suggestion
Add warning about ``--extra-index-url`` and dependency confusion
```

Otherwise `--` would become an en dash.

Reply:
No, it won't! But this is a good idea anyway.

https://github.com/pypa/pip/blob/master/docs/html/conf.py#L195